book/src/proposals/leader-duplicate-block-slashing.md

```diff
@@ -0,0 +1,99 @@
+# Leader Duplicate Block Slashing
+
+This design describes how the cluster slashes leaders that produce duplicate
```

Comment:
quick question: Well, if this is not a compliment to #7622, how bad leaders are punished/slashed? I mean, if all of validators are doing correct things (immediate discard of duplicate blocks and no duplicate votes even if voted), how validator's economic incentives are damaged other than merely decreased uptime?

Maybe we need a Duplicate Block Generation Proof advertisement in addition to flagging the block as "duplicate" locally?

Reply:
In other words, my concern here is that there may be no persisted indication of duplicate blocks?

Reply:
@ryoqun the leader would still get slashed. The "Proof of a Duplicate Shred" section in #7622 still applies here

Reply:
Oh, I see!